src/components/Annotation/Annotation.scss

```diff
@@ -0,0 +1,128 @@
+@import '../../common.scss';
+
+.ba {
+    .ba-annotation-container {
+        position: relative;
+        z-index: 1;
+
+        .ba-annotation-error {
+            padding: 14px 15px;
```

Comment:
For that values that are very specific (e.g. 14px, 23px) can you provide comments as to what calculation leads to those values? I'm thinking about going in to make changes to this CSS and it'd be great to know why certain things are the way they are.